e2e/tests/upgrades/genesis_test.go

```diff
@@ -32,20 +32,23 @@ func (s *GenesisTestSuite) TestIBCGenesis() {
 
 	configFileOverrides := make(map[string]any)
 	appTomlOverrides := make(test.Toml)
-	configFileOverrides["halt-height"] = haltHeight
-
-	testconfig.DefaultChainOptions().ChainAConfig.ConfigFileOverrides = appTomlOverrides
-
+	
+	appTomlOverrides["halt-height"] = haltHeight
+	configFileOverrides["config/app.toml"] = appTomlOverrides
+	chainOpts := func(options * testconfig.ChainOptions) {
+	    options.ChainAConfig.ConfigFileOverrides = configFileOverrides
+	}
+	
+	// create chains with specified chain configuration options
+	chainA, chainB := s.GetChains(chainOpts)
```

Comment:
I think this is the first time we'll have a test adding additional configuration to chain options, this code should work, if the options aren't being applied as expected I can help look into why 👍

Reply:
thank @chatton